corehq/apps/export/views/list.py

```diff
@@ -322,7 +322,7 @@ def get_saved_exports(self):
         exports = get_form_exports_by_domain(self.domain,
                                              self.permissions.has_deid_view_permissions,
                                              include_docs=False)
-        return [x for x in exports if not x['is_daily_saved_export']]
```

Comment:
Since this is shared across all `get_saved_exports` calls, you might move the sorting into `get_exports_page`.

Reply:
Good point, updated.